src/main/java/com/aliyuncs/fc/model/AccountSettings.java

```diff
@@ -0,0 +1,17 @@
+package com.aliyuncs.fc.model;
+
+/**
+ * Created by tw108174 on 18/10/30.
+ */
+public class AccountSettings {
+
+    private String[] availableAZs;
+
+    public String[] getAvailableAZs() {
+        return availableAZs;
+    }
+
+    public void setAvailableAZs(String[] availableAZs) {
```

Comment:
as discussed, we don't need to provide set function here

Reply:
done